bin/cmsl1t_dirty_batch

```diff
@@ -116,20 +116,41 @@ class cmsl1t_batch_job_bsub(object):
         return True
 
 
+def prepare_input_file_groups(input_ntuples, files_per_job):
+    file_lists = []
+    current_list = []
+    for infile in input_ntuples:
+        if not infile.startswith("root:"):
+            infile = os.path.realpath(infile)
+        current_list.append(infile)
+
+        # Is the current list full?
+        if len(current_list) >= files_per_job:
+            file_lists.append(current_list)
+            current_list = []
+
+    # Even if the last list had fewer files than needed, make sure to use this too
+    if current_list:
+        file_lists.append(current_list)
+
+    return file_lists
+
+
 @click.command()
 @click.option('-c', '--config_file', help='YAML style config file', type=click.File(), required=True)
+@click.option('-f', '--files-per-job', help='Give each job this many files', type=int, default=1)
 @click.option('--debug/--no-debug', help='Debug mode for the job submission', default=False)
 @click.option('--batch', default="bsub", type=click.Choice(["bsub", "htcondor"]),
               help='Select the job submission system to use')
-@click_log.simple_verbosity_option()
-@click_log.init(__name__)
-def run(config_file, debug, batch):
+@click_log.simple_verbosity_option(logger)
```

Comment:
Do you not need `click_log.basic_config(logger)` at the top of the file?
In other words, did you try to run it with the changes?

Reply:
Yes, I've tried it and it seems to work fine, with jobs being submitted and succeeding and output being logged.  I've not actually done anything with the verbosity option itself, however.